nilearn/datasets/tests/test_func.py

```diff
@@ -129,7 +129,8 @@ def test_fetch_haxby():
     for i in range(1, 6):
         haxby = func.fetch_haxby(data_dir=tmpdir, n_subjects=i,
                                  verbose=0)
-        assert_equal(len(mock_url_request.urls), 1 + (i == 1))  # subject_data + md5
+        assert_equal(len(mock_url_request.urls), 1 + (i == 1))
+                     # subject_data + md5
```

Comment:
I guess this comment should be before the assert now that it is not inline anymore